plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -462,7 +462,7 @@ lookupVarName varName varEnv = do
 -- | The entering point to the CEK machine's engine.
 enterComputeCek
     :: forall uni fun s
-    . (Ix fun, PrettyUni uni fun, GivenCekReqs uni fun s)
+    . (Ix fun, PrettyUni uni fun, GivenCekReqs uni fun s, uni `Everywhere` ExMemoryUsage)
     => CekMachineCosts
     -> Context uni fun
     -> CekValEnv uni fun
```

Comment:
So this still receives a `TermWithMem`? Can we drop `memTerm = withMemory term` from `runCek` and then erase all occurrences of `TermWithMem` completely?

Reply:
I was going to say "I'll do that separately"... but actually I think it is indeed totally dead now, so I might as well remove those annotations too.